README.md

```diff
@@ -6,7 +6,86 @@ This repo contains the pairing test exercises that are used in the Guardian recr
 
 Inspired to [work for us](http://developers.theguardian.com/join-the-team.html)?
 
-Interviewers, please read the [guidelines](https://docs.google.com/document/d/1ergle34ZDP5j8brTnhjnti7sMfLuykmY5E0c_6MS3bE).
+## What is it?
+The pairing test is the penultimate stage of the [interview process](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers) for the candidate.
+
+It is a 45-60 minute exercise where you and the candidate work as a pair writing code to solve a problem.
+
+Similar to when you pair with a colleague, there will be a driver and a navigator. The driver will be the one at the keyboard, whilst the navigator will be on the side making suggestions and asking questions.
+
+You’ll typically play the role of navigator.
+
+## Why do we perform it?
+[From: How does the Guardian recruit developers?](https://www.theguardian.com/info/developer-blog/2015/jan/20/how-does-the-guardian-recruit-developers)
+> I think pairing tests are the fairest form of interviewing you can offer. I know they can be stressful but they represent a big commitment in terms of effort and time. They create a situation that approximates the kind of work the organisation does rather than artificial trivia or whiteboard tests. They also give the candidate a chance to meet some of the people who already work at the Guardian and see if the environment suits them.
+
+The pairing test allows us to assess a candidate’s approach to solving a problem, what they prioritise, how they communicate their thinking and how they respond to any suggestions or advice. Ultimately, its a chance for us to assess what it would be like to work with the candidate and also a chance for the candidate to assess if the Guardian is a good fit for them.
+
+We’re not assessing a candidate’s pure technical ability and searching online is perfectly fine and encouraged.
+
+## What language?
+[From: Changing the Guardian's pairing test](https://www.theguardian.com/info/developer-blog/2014/may/28/changing-the-guardians-pairing-test)
+> ...allow candidates to do the pairing interview in the language of their choice. We would still prefer if candidates used our core languages of Scala, Python and JavaScript but if Haskell, Clojure, Ruby or Go are more your bag, then feel free.
+
+Whilst Scala or JavaScript, Swift/Objective C for iOS, Kotlin or Java for Android, are the preferred languages, a candidate can elect their own.
+
+The candidate should be the driver in the pairing exercise (especially so if you are not  familiar with the language).
```

Comment:
```suggestion
If the interviewer is not familiar with the language used, given the time available, the candidate would have to be the driver.
```

Reply:
The suggestion almost implies if the interviewer is familiar, they should drive. Not sure that's desired?

Reply:
Just felt like we're repeatin ourselves!  but the simple change is
```suggestion
The candidate should be the driver in the pairing exercise (especially so if the interviewer is not familiar with the language).
```

Reply:
Should we remove this line entirely then? All in favour of avoiding repetition.